modules/monitoring-silencing-alerts.adoc

```diff
@@ -1,30 +1,54 @@
 // Module included in the following assemblies:
 //
-// * monitoring/cluster_monitoring/managing-cluster-alerts.adoc
+// * monitoring/managing-alerts.adoc
 
-[id="monitoring-silencing-alerts_{context}"]
+[id="silencing-alerts_{context}"]
 = Silencing alerts
 
 You can either silence a specific alert or silence alerts that match a specification that you define.
 
+.Prerequisites
+
+* You have access to the cluster as a developer or as a user with `edit` permissions for the project that you are viewing metrics for.
+
 .Procedure
 
-To silence a set of alerts by creating an alert specification:
+To silence a specific alert:
 
-. Navigate to the *Monitoring* -> *Alerting* -> *Silences* page of the {product-title} web console.
+* In the Administrator perspective:
 
-. Click *Create Silence*.
+. Navigate to the *Monitoring* -> *Alerting* -> *Alerts* page of the {product-title} web console.
 
-. Populate the *Create Silence* form.
+. For the alert that you want to silence, select the {kebab} in the right-hand column and select *Silence Alert*. The *Create Silence* form will appear with a prepopulated specification for the chosen alert.
```

Comment:
The `Create Silence` form is fine, actually the form name is `Silence Alert`

Reply:
Thank you for highlighting this. Resolved.